CMakeLists.txt

```
@@ -8,8 +8,8 @@ project(ignition-plugin2 VERSION 2.0.0)
 #============================================================================
 # Find ignition-cmake
 #============================================================================
-find_package(ignition-cmake2 2.8.0 REQUIRED)
-set(IGN_CMAKE_VER ${ignition-cmake2_VERSION_MAJOR})
+find_package(ignition-cmake3 2.8.0 REQUIRED)
```

Same issue as on other source packages, the script is not smart enough here

```suggestion
find_package(ignition-cmake3 3.0.0 REQUIRED)
```